### PR TITLE
Makes it so when you lie down it makes you lie down in the direction you're facing

### PR DESCRIPTION
**Pull request by monster860**
## About The Pull Request

If you are facing down you lie down left, if you are facing right you lie down right

## Why It's Good For The Game

Imagine you want to lay down on a bed with someone. So, you hit rest. Oh shit, wrong direction! Time to get back up and hit rest again. Nope, its the wrong direction again. Time to get back up again. Hit rest again. GOD DAMN IT WHY IS IT THE WRONG FUCKING DIRECTION AGAIN JUST GIVE ME THE CORRECT DIRECTION HOW MANY TRIES COULD THIS POSSIBLY TAKE.

Okay, maybe you don't rage this hard, but it's still annoying to do this.

## Changelog
:cl:
tweak: You now lie down in the direction you are facing
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -1295,6 +1295,14 @@
...
 	if(!new_lying_angle)
+		//SKYRAT EDIT ADDITION BEGIN - SOUNDS
+		if(dir == WEST)
+			set_lying_angle(270)
+			return
+		else if(dir == EAST)
+			set_lying_angle(90)
+			return
+		//SKYRAT EDIT END
 		set_lying_angle(pick(90, 270))
```
Comment (Azarak) [inline code/modules/mob/living/carbon/carbon.dm:1306]: This always picks 90, right... it's gotta be a list. well guess thats an upstream issue

Comment (monster860) [reply code/modules/mob/living/carbon/carbon.dm:1306]: Pick can be a list or not a list

Comment (Azarak) [reply code/modules/mob/living/carbon/carbon.dm:1306]: interesting, I remember some issues from picks of non-lists

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -1295,6 +1295,14 @@
 /// Special carbon interaction on lying down, to transform its sprite by a rotation.
 /mob/living/carbon/proc/lying_angle_on_lying_down(new_lying_angle)
 	if(!new_lying_angle)
+		//SKYRAT EDIT ADDITION BEGIN - SOUNDS
```
Comment (Azarak) [inline code/modules/mob/living/carbon/carbon.dm]:
```suggestion
		//SKYRAT EDIT ADDITION BEGIN
```
Dont mark it into a module if it doesnt fit any, it's fine

**Review by Azarak — APPROVED** [review]

**Pull request merged**